channelserver/src/server.rs

```diff
@@ -237,6 +249,15 @@ impl Handler<Connect> for ChannelServer {
                 );
                 self.sessions.remove(&new_chan.id);
                 // self.metrics.borrow().incr("conn.max.conn").ok();
+                // Add the abuser to the abuse table?
+                /*
+                    // Not sure if I should add this sort of abuse.
+                    // Might penalize legit attempts to connect that are
+                    // delayed.
+                if let Some(remote) = msg.remote {
+                    self.abuser.add_abuser(&remote);
+                }
+                */
```

Comment:
I don't think we need to flag the IP as abusive here, especially not if it's their first time doing this which could be accidental.  More useful would be to log some sort of event which can be ingested and aggregated by the pipeline we were talking about earlier today, so that if an IP is *habitually* doing this, we can take some sort of action.

Reply:
Agreed, that's one of the reasons I have it commented out. One other possibility (if I understand the IPRep server correctly) is to impose a minor penalty on the address. Say -1 point for such an infraction. Infrequent IPs may get minor dings, where serial IPs would degrade much faster.

Reply:
"Minor infraction" seems reasonable to me, both here and in the other places you report abusive behaviour.